metrics/tikvclient.go

```diff
@@ -143,7 +134,7 @@ var (
 			Subsystem: "tikvclient",
 			Name:      "rawkv_kv_size_bytes",
 			Help:      "Size of key/value to put, in bytes.",
-			Buckets:   prometheus.ExponentialBuckets(1, 2, 21),
+			Buckets:   prometheus.ExponentialBuckets(1, 2, 21), // 1Byte ~ 2MB
```

Comment:
The max size of key plus value is 6MB. So I think this metric need larger bucket.

Reply:
addressed, PTAL @shenli